Mathlib/LinearAlgebra/SpecialLinearGroup.lean

```diff
@@ -0,0 +1,288 @@
+/-
+Copyright (c) 2025 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+
+import Mathlib.LinearAlgebra.Determinant
+import Mathlib.LinearAlgebra.Dual.Basis
+import Mathlib.LinearAlgebra.Matrix.Dual
+import Mathlib.LinearAlgebra.GeneralLinearGroup
+import Mathlib.LinearAlgebra.Charpoly.BaseChange
+
+/-!
+# The special linear group
+
+see also `Mathlib.LinearAlgebra.Matrix.SpecialLinearGroup`
```

Comment:
I don't think this is great. I have seen this in the general linear group and I don't think that is good either, especially since the link there is broken. I think we should have decent documentation in all files.